docker-compose.yml

```diff
@@ -0,0 +1,101 @@
+version: "3.8"
+
+services:
+  ingress-proxy:
+    container_name: dissolution-ingress-proxy
+    image: traefik:v2.2
+    command: --api.insecure=true --providers.docker
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock # So that Traefik can listen to the Docker events
+    networks:
+      - chs-dev
+    ports:
+      - 80:80
+      - 8080:8080 # Web UI (enabled by --api.insecure=true)
+  dissolution-web:
+    build: .
+    image: dissolution-web:1.0
+    labels:
+      - traefik.http.routers.lfp-appeals-frontend.rule=Host(`chs-dev`) && PathPrefix(`/dissolution-web/`)
+    ports:
+      - 3000:3000
+    networks:
+      - chs-dev
+    working_dir: /app
+    environment:
+      - CDN_HOST=cdn.chs-dev
+      - CACHE_SERVER=0.0.0.0:6379
+      - CACHE_DB=0
+      - CACHE_PASSWORD=''
+
+      - COOKIE_NAME=SID
+      - COOKIE_DOMAIN=chs-dev.internal
+      - COOKIE_SECURE_ONLY=0
+      - COOKIE_SECRET=ChGovUk-XQrbf3sLj2abFxIY2TlapsJ
+      - DEFAULT_SESSION_EXPIRATION=3600
+
+      - ACCOUNT_WEB_URL=http://web.chs-dev.internal:4000
+
+      # - FILE_TRANSFER_API_URL:
+      # - FILE_TRANSFER_API_KEY:
+      - MAX_FILE_SIZE_BYTES=4194304
+      - SUPPORTED_MIME_TYPES='application/pdf,image/jpeg,image/png,application/vnd.openxmlformats-officedocument.wordprocessingml.document,application/zip,application/octet-stream,application/x-zip-compressed,application/x-zip,multipart/x-zip,image/gif,application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'
+      - MAX_NUMBER_OF_FILES=10
+
+      - FILE_TRANSFER_FEATURE=1
+
+      - KAFKA_BROKER_ADDR=chs-kafka:9092
+
+      - SC_TEAM_EMAIL='appeals.ch.fake+SC@gmail.com'
+      - NI_TEAM_EMAIL='appeals.ch.fake+NI@gmail.com'
+      - DEFAULT_TEAM_EMAIL='appeals.ch.fake+DEFAULT@gmail.com'
+      # APPEALS_API_URL=http://api.chs-dev.internal
+      - LOG_LEVEL=info
+      - HUMAN_LOG=1
+  cdn-ch-gov-uk:
+    container_name: chs-cdn-ch-gov-uk
+    image: 169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/cdn.ch.gov.uk:latest
+    labels:
+      - traefik.http.routers.cdn-ch-gov-uk.rule=Host(`cdn.chs-dev`)
+    networks:
+      - chs-dev
+    expose:
+      - 80
+  account-ch-gov-uk:
```

Comment:
I think a redis container may also be needed to facilitate accounts. Damian has set some up on his POC. To test this, you should be able to access account.chs-dev/signin through the browser

Reply:
Ah ok, I'll integrate the redis part as well.